google-cloud-compute/src/main/java/com/google/cloud/compute/ComputeOptions.java

```diff
@@ -49,7 +50,7 @@ public Compute create(ComputeOptions options) {
     private static final ComputeRpcFactory INSTANCE = new DefaultComputeRpcFactory();
 
     @Override
-    public ComputeRpc create(ComputeOptions options) {
+    public ServiceRpc create(ComputeOptions options) {
```

Comment:
It's not that we can't; the problem is that the `ComputeRpc` interface is tied to a specific major version of compute, and if we want to swap it out later for a different major version, that would be a breaking change. However, if the return type is `ServiceRpc`, we can swap it out without breaking direct callers.